modules/dynamic-provisioning-defining-storage-class.adoc

```diff
@@ -9,6 +9,7 @@
 `StorageClass` objects are currently a globally scoped object and must be
 created by `cluster-admin` or `storage-admin` users.
 
+ifndef::microshift[]
```

Comment:
I know this is going to be a pain, but you should really go through all of these modules you're reusing and add the appropriate microshift assembly to the list of assemblies the module is included in. :grin: